.github/workflows/e2e-tests.yml

```diff
@@ -0,0 +1,109 @@
+name: E2E Tests
```

Comment:
@shaeed-noordeen Let's remove the e2e tests from being run in GHA and leave them in TC for now -- we plan on removing these tests altogether in favor of application specific testing which is underway this release.